<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Budget Track</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(3, 156, 36);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="acctTotalLabel">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>48</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Acct Total</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="acctTotalValue">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>24</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>$0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="showTransactionsButton">
        <property name="text">
         <string>Transactions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showCCButton">
        <property name="text">
         <string>Credit Debt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showBillsButton">
        <property name="text">
         <string>Current Bills</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addTransButton">
        <property name="text">
         <string>Add Transaction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="modifyTransButton">
        <property name="text">
         <string>Modify Transaction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateCCTotalsButton">
        <property name="text">
         <string>Update CC Totals</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="modifyBillsButton">
        <property name="text">
         <string>Modify Bills</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>48</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Fixed Total</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>24</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>$0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1073</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
